apps/console/src/features/connections/components/edit/settings/attribute-management/uri-attributes-settings.tsx

```diff
@@ -94,6 +95,154 @@ export const UriAttributesSettings: FunctionComponent<AdvanceAttributeSettingsPr
+                                    "console:develop.features.authenticationProvider." +
+                                        "forms.uriAttributeSettings.group.message"
+                                }
+                                tOptions={ {
+                                    attribute: isOIDC
+                                        ? ConnectionManagementConstants.OIDC_ROLES_CLAIM
+                                        : ConnectionManagementConstants.CLAIM_ROLES,
+                                    suffix: isOIDC
+                                        ? "OIDC attribute"
+                                        : "attribute"
+                                } }
+                            >
+                                Please note that <strong>{ isOIDC
+                                    ? ConnectionManagementConstants.OIDC_ROLES_CLAIM
+                                    : ConnectionManagementConstants.CLAIM_ROLES }</strong>
+                                will be considered as the default
+                                <strong> Group Attribute</strong> as you have not added a
+                                custom attribute mapping.
+                            </Trans>
+                        )
+                    }
+                />
+            );
+        }
+
+        // Custom attribute mapping is enabled
+        // roleClaimURI is not empty.
+        // No need to show the info box as there is a custom attribute mapping.
+        if (!isEmpty(initialRoleUri)) {
+            return null;
+        }
+
+        // Custom attribute mapping is enabled
+        // roleClaimURI is empty.
+        if (UIConfig.isCustomClaimMappingEnabled && !UIConfig.isCustomClaimMappingMergeEnabled) {
+            const mappedRolesClaim: string = resolveMappedRolesClaimAttributeName();
+
+            // roles mapped attribute present
+            if (!isEmpty(mappedRolesClaim)) {
+                return (
+                    <Message
+                        type="info"
+                        content={
+                            (
+                                <Trans
+                                    i18nKey={
+                                        "console:develop.features.authenticationProvider." +
+                                            "forms.uriAttributeSettings.group.mappedRolesPresentMessage"
+                                    }
+                                    tOptions={ {
+                                        mappedRolesClaim: mappedRolesClaim
+                                    } }
+                                >
+                                    Please note that <strong>{ mappedRolesClaim }</strong>
+                                    will be considered as the default
+                                    <strong> Group Attribute</strong> with the current configuration.
+                                    You can select an attribute from the dropdown.
```

Comment:
Feels like this last line might be redundant since we already have a hint for the field. WDYT?

Reply:
Since these were finalized with @AnuradhaSK and @SujanSanjula96. Let's keep it as it for now.